vector-stores/spring-ai-redis-store/src/main/java/org/springframework/ai/vectorstore/redis/RedisFilterExpressionConverter.java

```diff
@@ -38,11 +38,11 @@
  */
 public class RedisFilterExpressionConverter extends AbstractFilterExpressionConverter {
 
-	public static final NumericBoundary POSITIVE_INFINITY = new NumericBoundary(Double.POSITIVE_INFINITY, true);
```

Comment:
These public constants cause visibility issues since the underlying types are package-protected.